src/_designs/screens/dashboard/index.vue

```diff
@@ -15,27 +20,62 @@
 <script>
 import Generations from '@/_designs/components/generations'
 import Generation from '@/_designs/components/generation'
-import PageHeader from '@/_designs/components/PageHeader'
-import BreadCrumbs from '@/_designs/components/BreadCrumbs'
 import { mapState } from 'vuex'
+import Chart from 'chart.js'
+import transactionRate from './transactionRate'
+import aeTransacted from './aeTransacted'
 
 export default {
   name: 'AppDashboard',
   components: {
-    PageHeader,
     Generations,
-    Generation,
-    BreadCrumbs
+    Generation
+  },
+  data () {
+    return {
+      transactionRate: transactionRate,
+      aeTransacted: aeTransacted
+    }
   },
   computed:
     mapState('blocks', [
       'generations'
     ]),
   mounted: async function () {
+    this.createChart('transaction-rate', this.transactionRate)
+    this.createChart('ae-transacted', this.aeTransacted)
     await this.$store.dispatch('blocks/getLatestGenerations', 5)
+  },
+  methods: {
+    createChart (chartId, chartData) {
+      const ctx = document.getElementById(chartId)
+      return new Chart(ctx, {
+        type: chartData.type,
+        data: chartData.data,
+        options: chartData.options
+      })
+    }
   }
 }
 </script>
 <style lang="scss">
-.app-dashboard {}
+@import "~@aeternity/aepp-components-3/src/styles/fallback/mixins";
+.app-dashboard {
+  .graphs-container{
+    display: flex;
+    justify-content: space-evenly;
+    flex-wrap: wrap;
+
+    .graph{
+      width: 22em;
+      height: 11em;
+      margin: 2em 1em 1em 0;
+      @include phone {
```

Comment:
We might use similar mixins later. But lets leave it native for now. Or we can start a discussion on this.